cockroachcloud/serverless-movr.md

```diff
@@ -0,0 +1,125 @@
+get-the-cluster-connection-string---
+title: Estimate Serverless Usage with MovR
+summary: Run the MovR sample application to estimate RU usage in CockroachDB Serverless.
+toc: true
+referral_id: docs_serverless_movr
+docs_area: get_started
+---
+
+This tutorial shows you how to initialize and run the MovR workload on a {{ site.data.products.serverless }} cluster, then use the Request Unit consumption data to estimate a sufficient budget for the MovR workload.
```

Comment:
Would be good to make 'Request Unit' link to the RU definition in the FAQ: https://cockroachlabs.com/docs/cockroachcloud/serverless-faqs.html#what-is-a-request-unit

Reply:
not for this PR necessarily but would be good IMO to link that FAQ entry on RUs to https://cockroachlabs.com/docs/cockroachcloud/plan-your-cluster.html#cluster-scaling , which is also helpful for understanding RUs, I think